plugin.js

```diff
@@ -80,6 +80,13 @@ function authenticate (keys, key) {
   return keys.findIndex((a) => compare(a, key)) !== -1
 }
 
+function compare (a, b) {
+  if (typeof a !== 'string' || typeof b !== 'string') return false
+  if (a.length !== b.length) return false
```

Comment:
Wouldn’t this be covered in the timing safe compare?

Reply:
Yes, but it throws `RangeError [ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH]: Input buffers must have the same length`

Reply:
So try/catch the error? Seems it’d be better to do the full comparison via the timing safe mechanism.

Reply:
It seems Node core also checks its `byteLength` before comparison. 👀 
https://github.com/nodejs/node/blob/master/lib/internal/crypto/util.js#L79-L92

Leave everything to Node is better..?

```js
function compare (a, b) {
  try {
    return crypto.timingSafeEqual(Buffer.from(a), Buffer.from(b))
  } catch {
    return false
  }
}
```

Reply:
Yep. Thanks.

Reply:
Fixed at ee397b7